services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -180,6 +182,11 @@ public void initP2PNetwork() {
     }
   }
 
+  public void initRestAPI() {
+    STDOUT.log(Level.DEBUG, "BeaconChainController.initRestAPI()");
+    new RestApi(chainStorageClient, p2pNetwork, 5051);
```

Comment:
This port should be a new config.toml param

Reply:
agreed